src/main/asciidoc/index.adoc

```diff
@@ -11,7 +11,9 @@ Sabby Anandan; David Turanski; Glenn Renfro; Eric Bottard; Mark Pollack; Chris S
 
 :spring-cloud-stream-docs: http://docs.spring.io/spring-cloud-stream/docs/{scst-core-version}/reference/htmlsingle/index.html
 :github-code: https://github.com/spring-cloud/spring-cloud-dataflow-samples
-
+:scdf-release-train-name: Bacon
+:app-import-kafka-maven: http://bit.ly/{scdf-release-train-name}-RELEASE-stream-applications-kafka-10-maven
```

Comment:
There's a catch with `{scdf-release-train-name}`.

Though the release-train is `Bacon`, `Celsius` etc., the following suffix could be `RELEASE`, `SRx`, or `BUILD-SNAPSHOT`. Ideally, we won't update the samples to use BS, but it is possible we will have to update it to SRx releases.

For example, stream apps are currently at: `Celsius.SR1`. 

Perhaps we can parameterize both the words in a single token? How about:

> scdf-release-train: Celsius.SR1
> :app-import-kafka-maven: http://bit.ly/{scdf-release-train}-stream-applications-kafka-10-maven

Reply:
That seems simple